tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,72 @@
+## **Create New Identity With Attributes Using Local Fabric CA**
+`5-10 mins`
+
+> **Important:** You should have a **smart contract** instantiated on your Local Fabric to follow this tutorial. Follow part 1 of this series first for instructions.
+
+
+## Learning Objectives
+
+* Create a new identity (with attributes) using Local Fabric CA.
+* Using the newly created identity and implement attribute-based action control.
+
+---
+<details>
+<summary><b>1. Creating a new identity with attributes</b></summary>
+
+In the very first tutorial (Tutorial 1: Local smart contract development), we used a pre-set identity (`admin`) to connect to the Local Fabric gateway and submit/evaluate transactions. In this tutorial, we will learn how to create _our own_ identity with attributes.
+
+Having multiple identities enrolled with the Local Fabric CA (Certificate Authority) allows you to represent different participants in a blockchain network. For example two identities `Tesla` (the manufacturer) and `Jack` (the customer) may be permitted to submit different transactions in a smart contract.
+
+Created identities can be assigned attributes, which allows attribute-based access control. A transaction function could be written such that if the value of a given attribute on the identity being used matches a particular value, it will do one thing; if the value does not match, it could do something else, or throw an error. How to write such logic into smart contract transactions will be covered in a later tutorial.
```

Comment:
doing this isn't covered in a later tutorial you have done it in this one

Reply:
Fixed